docs/installation.rst

```diff
@@ -14,15 +14,19 @@ Configuration
 ---------------
 
 
-Add ``djstripe`` to your ``INSTALLED_APPS``:
+Add ``djstripe`` to your ``INSTALLED_APPS``. You will also need the `sites` framework enabled.:
```

Comment:
Hmm you're right. I'll create an issue to get a consensus on removing it. For now, this is fine.